packages/shared/lib/core/network/utils/buildChainFromNetwork.ts

```diff
@@ -0,0 +1,13 @@
+import { selectedAccount } from '@core/account/stores'
+import { activeProfile } from '@core/profile'
```

Comment:
Might be good to import from `@core/profile/stores` to help out with weird circular deps later on